docs/link_handling.md

```diff
@@ -48,15 +48,8 @@ Sample iOS universal link metadata
 	"applinks": {
 		"details": [
```

Comment:
According to the docs the following key must exist:
```json
"apps": []
```
From the docs
> The apps key in an apple-app-site-association file must be present and its value must be an empty array
https://developer.apple.com/library/archive/documentation/General/Conceptual/AppSearch/UniversalLinks.html